mailpoet/lib/API/JSON/v1/Settings.php

```diff
@@ -493,4 +495,25 @@ public function reactivateReEngagementEmails(): array {
       'action' => 'reactivate',
     ];
   }
+
+  /**
+   * Prepares the settings to set up MSS with the given key and calls the set method.
+   *
+   * @param string $apiKey
+   * @return ErrorResponse|SuccessResponse
+   */
+  public function setupMSS(string $apiKey) {
+    $new_settings = [
+      'mta_group' => 'mailpoet',
+      'mta' => [
+        'method' => 'MailPoet',
```

Comment:
We have a PHP constant for that `MailPoet\Mailer\Mailer::METHOD_MAILPOET` so please use the constant.

Reply:
Thanks and done.